flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/HsSubpartitionMemoryDataManager.java

```diff
@@ -359,13 +364,29 @@ private void finishCurrentWritingBuffer() {
         Buffer buffer = bufferConsumer.build();
         currentWritingBuffer.close();
         bufferConsumer.close();
-
+        // TODO support buffer compression
```

Comment:
What is this TODO for?

Reply:
Good catch, I forget to remove this TODO comment.